rebar.config

```diff
@@ -1,5 +1,6 @@
 {deps, 
   [
+   {edown, ".*", {git, "https://github.com/uwiger/edown.git", {branch, "master"}}},
```

Comment:
`edown` doesn't look like direct dependency of seresye. It is in fact the dependency of `parser_trans`.

Instead I propose more kosher approach: to change the `parse_trans` repo to: `{parse_trans, ".*", {git, "https://github.com/uwiger/parse_trans.git", {branch, "master"}}}`. Moreover `esl/parse_trans` fork is significantly outdated (22 commits behind uwiger:master).

Apart from above: :+1: